go/src/koding/kites/kloud/provider/google/stack.go

```diff
@@ -359,6 +359,23 @@ func (s *Stack) getDiskSize(currentProject string, computeService *compute.Servi
 	}
 }
 
+func flatten(data []map[string]interface{}) map[string]interface{} {
+	res := make(map[string]interface{})
+	for _, ms := range data {
+		for key, m := range ms {
+			if val, ok := m.(string); ok {
+				if res[key] == nil {
+					res[key] = val
+				} else {
+					res[key] = res[key].(string) + "\n" + val
```

Comment:
Why do you merge with newline? Is it possible to specify in terraform template the same key multiple times with different values?